src/installing-workflow/system-requirements.md

```diff
@@ -26,15 +26,6 @@ application footprint as well.
 Running smaller machines will likely result in increased system load and has been known to result in component failures
 and instability.
 
-## Docker Insecure Registry
```

Comment:
We might want to keep the warning that "versions prior to 2.2 require '--insecure-registry' to function properly".   To keep our bases covered.